examples/manifests/route.yaml

```diff
@@ -3,6 +3,7 @@ apiVersion: route.openshift.io/v1
 kind: Route
 metadata:
   name: hw-event-proxy
+  namespace: cloud-native-events
```

Comment:
do you need this route.yaml here ? it should be under hw-event-proxy

Reply:
deleted